src/services/mongoose/accounts.ts

```diff
@@ -171,7 +185,7 @@ export const AccountsRepository = (): IAccountsRepository => {
   }
 
   return {
-    persistNewKratosUser,
+    persistNew,
```

Comment:
uhmm so now user = account?

Reply:
`User` from mongoose = `Account` domain object (mongoose collection to be renamed in follow up PR)